pkg/reconciler/clusteringress/clusteringress.go

```diff
@@ -161,7 +161,7 @@ func (c *Reconciler) reconcile(ctx context.Context, ci *v1alpha1.ClusterIngress)
 	logger.Infof("Reconciling clusterIngress: %#v", ci)
 
 	gatewayNames := gatewayNamesFromContext(ctx, ci)
-	vses := resources.MakeVirtualServices(ci, gatewayNames)
+	vses := resources.MakeVirtualServicesForClusterIngress(ci, gatewayNames)
```

Comment:
I'm puzzled by the abundance of `*ForClusterIngress` from a `resources` package scoped to cluster ingress. 🤔

Reply:
Once ClusterIngress codes are retired, al these ForClusterIngress/ForIngress can be cleaned up..